tt_metal/hw/inc/dev_msgs.h

```diff
@@ -49,25 +49,30 @@ enum dispatch_mode {
     DISPATCH_MODE_HOST,
 };
 
+enum dispatch_core_processor_classes {
+    // Tensix processor classes
+    DISPATCH_CLASS_TENSIX_DM0 = 0,
+    DISPATCH_CLASS_TENSIX_DM1 = 1,
+    DISPATCH_CLASS_TENSIX_COMPUTE = 2,
+
+    // Ethernet processor classes
+    DISPATCH_CLASS_ETH_DM0 = 0,
+
+    DISPATCH_CLASS_COUNT = 3,
```

Comment:
Only confusing part for me is when `DISPATCH_CLASS_COUNT` gets used on it's own, I don't immediately recognize that it's referring to all available processor types, maybe include `CORE` in there somewhere? If it makes sense to everyone else then not an issue though

Reply:
funny, I had CORE there but thought it was confusing due to CORE_COUNT sounding like "# tensix cores".  I then tried PROC or PROCESSOR.  What if "count" was "max", would that help?

Reply:
maybe `DISPATCH_CLASS_TENSIX_PROC_COUNT` little pedantic though

Reply:
I think it should have MAX in it since it is technically the MAX(#tensix classes, #eth classes).  I don't like using both count and max.  Haven't found anything I like.  Is this odd: DISPATCH_CLASS_MAX_CLASSES?  Or maybe just DISPATCH_CLASS_MAX_PROC?  Leaning toward the latter

Reply:
DISPATCH_CLASS_MAX_PROC > DISPATCH_CLASS_MAX_CLASSES I think as well

Reply:
DISPATCH_CLASS_MAX_PROC it is